component/snackbar/snackbar.html

```diff
@@ -0,0 +1,145 @@
+    <link rel="stylesheet" href="/styles.css" />
+    <link rel="stylesheet" href="/component/button/button.css" />
+
+    <title>UiPlugIn | Snackbar</title>
+  </head>
+  <body>
+    <!-- NavBar -->
+
+    <nav class="navigation container">
+      <i class="nav-icon fab fa-uikit"></i>
+      <div class="nav-brand">UiPlugIn</div>
+      <ul class="list-non-bullet nav-pills">
+        <li class="list-item-inline">
+          <a class="link" href="/index.html">Home</a>
+        </li>
+
+        <li class="list-item-inline">
+          <a class="link" href="/docs.html">Documentation</a>
+        </li>
+      </ul>
+    </nav>
+    <!-- Navbar -->
+    <main class="main-content">
+      <!-- Sidebar -->
+      <aside class="side-nav">
+        <p class="sidebar-heading">Components</p>
+        <ul class="sidebar-list">
+          <li><a href="/component/avatar/avatar.html">Avatars</a></li>
+          <li>
+            <a href="/component/badge/badge.html">Badges</a>
+          </li>
+          <li><a href="/component/alert/alert.html">Alerts</a></li>
+          <li><a href="/component/button/button.html">Buttons</a></li>
+          <li><a href="/component/card/card.html">Cards</a></li>
+          <li><a href="/component/image-responsive/image.html">Images</a></li>
+          <li><a href="/component/input/input.html">Inputs</a></li>
+          <li><a href="/component/text/text-utility.html">Text</a></li>
+          <li><a href="/component/lists/list.html">Lists</a></li>
+          <li><a href="/component/navigation/navigation.html">Navigation</a></li>
+          <li><a href="/component/slider/slider.html">Range Sliders</a></li>
+          <li><a href="/component/ratings/ratings.html">Ratings</a></li>
+          <li><a href="/component/grid/grid.html">Grid</a></li>
+          <li><a href="/component/snackbar/snackbar.html" class="active">Snackbar</a></li>
+        </ul>
+      </aside>
+
+      <div class="doc-content">
+        <h1 class="snackbar-title">Snackbar/Toast/Notify</h1>
+        <p class="snackbar-description">
+          A snackbar is used to show a message at the bottom of the screen.
+        </p>
+        <!-- Notification -->
+        <button onclick="myFunction()" class="btns primary">
+          Show Snackbar
+        </button>
+        <div id="snackbar">Welcome to UiPlugIn</div>
+
+        <!-- Code -->
+        <div class="code">
+          <pre>
+            <code class="html">
+                <button onclick="myFunction()">Show Snackbar</button>
+                <div id="snackbar">Welcome to UiPlugIn</div>
+                
+                <!-- JavaScript -->
+                <script>
+                    function myFunction() {
+                      const x = document.getElementById("snackbar");
+                      x.className = "show";
+                      setTimeout(function () {
+                        x.className = x.className.replace("show", "");
+                      }, 3000);
+                    }

```

Comment:
How about adding it to separate js file. But its ok for now to do this.

Reply:
will work on this, thanks